treeherder/model/migrations/0055_autoclassify_status.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+# Original SQL:
+#
+# BEGIN;
+# --
+# -- Add field autoclassify_status to job
+# --
+# ALTER TABLE `job` ADD COLUMN `autoclassify_status` integer DEFAULT 0 NOT NULL;
+# ALTER TABLE `job` ALTER COLUMN `autoclassify_status` DROP DEFAULT;
+# COMMIT;
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('model', '0054_duplicate_failure_classifications'),
+    ]
+
+    operations = [migrations.RunSQL(
+        sql="""
+--
+-- Add field autoclassify_status to job
+--
+SET FOREIGN_KEY_CHECKS=0;
+ALTER TABLE `job` ADD COLUMN `autoclassify_status` integer DEFAULT 0 NOT NULL;
+ALTER TABLE `job` ALTER COLUMN `autoclassify_status` DROP DEFAULT;
```

Comment:
The merged statement shouldn't include both the `ADD COLUMN` and `ALTER COLUMN` parts, which would fix that error.

However changing defaults is a metadata only change, so no performance benefit from combining, so to avoid increasing the divergence from what Django auto-generates (which increased the risk of mistakes), I'm fine with leaving as-is.